intro-HPC/ch_preparing_the_environment.tex

```diff
@@ -189,7 +189,7 @@ \section{First Time connection to the \hpc}
   \subsection{Connect}
   \label{sec:connect}
 
-  The \hpc is only accessible from within the \university network, but you can
+  At some institutes the \hpc is only accessible from within the \university network, but you can
```

Comment:
Is this still true? I know for us  (VUB) it is not the case, Leuven neither. Gent the same? I think only Antwerp might be? @klust ?

Reply:
Hence the change in wording here.
I wanted to add an `\ifantwerp`, but it seems to break inside an 
`\ifmacorlinux`